help/follow-a-topic.md

```diff
@@ -0,0 +1,32 @@
+# Follow a topic
+
+Zulip lets you follow topics you are interested in, and configure what
+notifications you'd like to receive for topics you follow.
+
+In muted streams, topics you follow are also automatically treated as
+[unmuted](/help/mute-a-topic).
+
+## Configure topic notifications
+
+{!configure-topic-notifications.md!}
+
+## Set notifications for followed topics
```

Comment:
If we go with the suggestion for the first subheading, maybe we should say "Configure" here instead:

```suggestion
## Configure notifications for followed topics
```

Reply:
I used "Set" to match https://zulip.com/help/stream-notifications. Should we change it to "Configure" both here and on that page, perhaps?

Reply:
Yeah, I think we should change it to "Configure" on both of these pages.